src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -808,17 +818,25 @@ private void ImportCall(ILOpcode opcode, int token)
             {
                 throw new NotImplementedException();
             }
-            if (opcode == ILOpcode.callvirt && callee.IsAbstract)
-            {
-                throw new NotImplementedException();
-            }
 
             if (callee.OwningType.IsDelegate)
             {
-                throw new NotImplementedException();
+                if (opcode == ILOpcode.newobj)
+                {
+                    DelegateCreationInfo delegateInfo = _compilation.GetDelegateCtor(callee.OwningType, ((LdTokenEntry<MethodDesc>)_stack.Peek()).LdToken, followVirtualDispatch: false);
```

Comment:
Add a `WASMTODO` to keep track of what to put in the `followVirtualDispatch` parameter? (This depends on whether `ldvirtftn` was used to load the target method.)

Reply:
Added tracking to the FunctionPointer entry and passed the value here